src/plugins/intel_gna/src/gna_graph_compiler.cpp

```diff
@@ -233,8 +237,11 @@ void GNAPluginNS::GNAGraphCompiler::ValidateCnn2D(const std::string& name,
                                                   const uint32_t dilW,
                                                   OvGnaType inPrecision) const {
     if (cnn2dValidator) {
-        cnn2dValidator
-            ->ValidateCnn2D(name, inHeight, inWidth, inChannels, kH, kW, kN, strideH, strideW, dilH, dilW, inPrecision);
+        if (use_only_cnn2D_iface && inHeight == 1 && kH == 1 && inChannels == 1) {
```

Comment:
Cannot we use !GNAConvolutionLayer::isConv2D here?

Reply:
I changed name to more precision for GNAConvolutionLayer::isConv2D -> GNAConvolutionLayer::is3DInputOr2DKernel. Additionally new method GNAConvolutionLayer::isConv1 is added and used here instead of GNAConvolutionLayer::isConv2